walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,13 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    String fixedName = "";
+    for(int i = 0; i < name.length(); i++){
```

Comment:
`String` also has a useful `trim()` method that does a similar thing, although it will only remove leading/trailing spaces. Your solution removes all spaces, even if they appear in the middle of the string.

Reply:
That's handy! I made the assumption that we were supposed to solve this without using any java functions, which is why i decided to just brute-force my way through it lol